<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>810</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>RIPHelper</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <widget class="QPushButton" name="start_stop_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #e4cfe0;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #e4cfe0;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="scans_frequency_box">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #ececec;
color: #5d0f4a;</string>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #5d0f4a;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   Folders to scan</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #5d0f4a;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   Scans frequency</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>881</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #ececec;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #5d0f4a;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>   Program path</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>410</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #e4cfe0;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color:#e4cfe0;
color:#5d0f4a;</string>
   </property>
   <property name="text">
    <string>Reset configurations</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_field">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>130</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #ececec;
color: #5d0f4a;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>841</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #5d0f4a;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="folders_box">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #916f86;
border-top-left-radius :10px;
border-top-right-radius : 10px; 
border-bottom-left-radius : 10px; 
border-bottom-right-radius : 10px;
color: #5d0f4a;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="add_folder_button">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>250</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #e4cfe0;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string>Add folder</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="backups_box">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>240</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #916f86;
border-top-left-radius :10px;
border-top-right-radius : 10px; 
border-bottom-left-radius : 10px; 
border-bottom-right-radius : 10px;
color:#5d0f4a;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_folder_button">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>290</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #e4cfe0;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string>Remove folder</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>1061</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #ececec;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #ececec;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string>   Logs</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="logs_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>511</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #916f86;
border-top-left-radius :10px;
border-top-right-radius : 10px; 
border-bottom-left-radius : 10px; 
border-bottom-right-radius : 10px;
color: #5d0f4a;</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustIgnored</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>200</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #5d0f4a;
color: #ececec;</string>
   </property>
   <property name="text">
    <string>   Backup folders</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>161</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #ececec;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: #ececec;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string>   Scanned files</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="disabled_view">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>881</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #ececec;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="disabled_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>861</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color: #e4cfe0;
color: #5d0f4a;</string>
   </property>
   <property name="text">
    <string> You may not edit configuration while the scan is running!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="scanned_files_box">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>520</y>
     <width>511</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #916f86;
border-top-left-radius :10px;
border-top-right-radius : 10px; 
border-bottom-left-radius : 10px; 
border-bottom-right-radius : 10px;
color: #5d0f4a;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_logs_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>490</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color:#e4cfe0;
color:#5d0f4a;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_scanned_files_button">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>490</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 3px solid #916f86;
border-top-left-radius :20px;
border-top-right-radius : 20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
background-color:#e4cfe0;
color:#5d0f4a;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <zorder>disabled_label</zorder>
  <zorder>disabled_view</zorder>
  <zorder>graphicsView</zorder>
  <zorder>start_stop_button</zorder>
  <zorder>label</zorder>
  <zorder>scans_frequency_box</zorder>
  <zorder>label_2</zorder>
  <zorder>browse_button</zorder>
  <zorder>save_button</zorder>
  <zorder>clear_button</zorder>
  <zorder>path_field</zorder>
  <zorder>graphicsView_2</zorder>
  <zorder>label_3</zorder>
  <zorder>folders_box</zorder>
  <zorder>add_folder_button</zorder>
  <zorder>backups_box</zorder>
  <zorder>remove_folder_button</zorder>
  <zorder>graphicsView_3</zorder>
  <zorder>label_4</zorder>
  <zorder>logs_box</zorder>
  <zorder>label_5</zorder>
  <zorder>graphicsView_4</zorder>
  <zorder>label_6</zorder>
  <zorder>scanned_files_box</zorder>
  <zorder>clear_logs_button</zorder>
  <zorder>clear_scanned_files_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
